nginx-nodejs-redis/web/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node:14.17.3-alpine3.14
+
+WORKDIR /usr/src/app
+
+COPY ./package.json ./
+RUN npm install
```

We should add the `package-lock.json` before installing the dependencies, using `npm ci` is also a best practise when building an image.

```suggestion
COPY package.json package-lock.json ./
RUN npm ci
```